src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -1036,6 +1037,7 @@ export async function activateRoslynLanguageServer(
     );
 
     registerLanguageStatusItems(context, languageServer, languageServerEvents);
+    await registerCopilotExtensionAsync(languageServer, _traceChannel);
```

Comment:
I'd recommend not using the trace channel - that is generally just for LSP trace logs.  What it should do is write to the normal output channel only when trace logging is enabled

Reply:
Done!